packages/task/src/browser/quick-open-task.ts

```diff
@@ -322,6 +307,31 @@ export class QuickOpenTask implements QuickOpenModel, QuickOpenHandler {
         });
     }
 
+    private addTaskItemsByTaskScope(groupedTasks: Map<string, TaskConfiguration[]>, token: number, isMulti: boolean, isFirstGroup: boolean, taskScopeString: string): boolean {
+        let showBorder = isFirstGroup ? false : true;
+        if (groupedTasks.has(taskScopeString)) {
+            const configs = groupedTasks.get(taskScopeString)!;
+            this.items.push(
+                ...configs.map(taskConfig => {
+                    const item = new TaskConfigureQuickOpenItem(
+                        token,
+                        taskConfig,
+                        this.taskService,
+                        this.taskNameResolver,
+                        this.workspaceService,
+                        isMulti,
+                        { showBorder: showBorder }
```

Comment:
minor: it can be omitted since the variable has the same name as the option:

```suggestion
                        { showBorder }
```

Reply:
No longer applicable in the latest commit, I will keep it in mind for next changes, thanks !